Loop/Views/SettingsView.swift

```diff
@@ -211,28 +213,41 @@ extension SettingsView {
     
     private var deletePumpDataSection: some View {
         Section {
-            Button(action: { self.viewModel.pumpManagerSettingsViewModel.deleteData?() }) {
+            Button(action: { self.showDeletePumpDataAlert.toggle() }) {
                 HStack {
                     Spacer()
                     Text("Delete Pump Data").accentColor(.destructive)
                     Spacer()
                 }
             }
+            .alert(isPresented: $showDeletePumpDataAlert) {
+                makeDeleteAlert(for: self.viewModel.pumpManagerSettingsViewModel)
+            }
         }
     }
     
     private var deleteCgmDataSection: some View {
         Section {
-            Button(action: { self.viewModel.cgmManagerSettingsViewModel.deleteData?() }) {
+            Button(action: { self.showDeleteCGMDataAlert.toggle() }) {
                 HStack {
                     Spacer()
                     Text("Delete CGM Data").accentColor(.destructive)
                     Spacer()
                 }
             }
+            .alert(isPresented: $showDeleteCGMDataAlert) {
+                makeDeleteAlert(for: self.viewModel.cgmManagerSettingsViewModel)
+            }
         }
     }
     
+    private func makeDeleteAlert(for model: DeviceViewModel) -> SwiftUI.Alert {
+        return SwiftUI.Alert(title: Text("Delete \(model.name()) Data"),
+                             message: Text("Are you sure you want to delete all your \(model.name()) Data?\n(This action is not reversible)"),
```

Comment:
This previously said "Are you sure you want to delete testing pump health data?" That makes it clear we're just deleting the simulator data, which is what happens here. I'm not sure what model.name() evaluates to now. Is it clear that it's testing data?

Reply:
Yes, it was for exactly that reason I did this: to be clear about what device's data is being deleted.  `model.name()` is exactly what's shown in device button, as reported by the vended UI (i.e. "Insulin Pump Simulator" or "CGM Simulator").  So, it would show 
> Are you sure you want to delete all your Insulin Pump Simulator Data?
![Simulator Screen Shot - iPhone SE (2nd generation) - 2020-09-01 at 09 56 54](https://user-images.githubusercontent.com/308993/91883634-8d87ce00-ec39-11ea-9a1d-865264745277.png)

Does that help?

Reply:
Yes, that looks great.